Sources/NIOPosix/MultiThreadedEventLoopGroup.swift

```diff
@@ -138,25 +139,48 @@ public final class MultiThreadedEventLoopGroup: EventLoopGroup {
     /// - arguments:
     ///     - numberOfThreads: The number of `Threads` to use.
     public convenience init(numberOfThreads: Int) {
-        self.init(numberOfThreads: numberOfThreads, selectorFactory: NIOPosix.Selector<NIORegistration>.init)
+        self.init(numberOfThreads: numberOfThreads,
+                  canBeShutDown: true,
+                  selectorFactory: NIOPosix.Selector<NIORegistration>.init)
+    }
+
+    public convenience init(_canBeShutDown canBeShutDown: Bool, numberOfThreads: Int) {
```

Comment:
Is this ever used with `_canBeShutDown: true`? A `static func` might be better here as you can underscore the method name (`init` with `_` never feels private enough to me)

Reply:
great idea, thanks, done